src/main/java/com/blackducksoftware/integration/hub/alert/channel/ChannelRestFactory.java

```diff
@@ -0,0 +1,61 @@
+package com.blackducksoftware.integration.hub.alert.channel;
+
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+
+import com.blackducksoftware.integration.exception.IntegrationException;
+import com.blackducksoftware.integration.hub.rest.RestConnection;
+import com.blackducksoftware.integration.hub.rest.exception.IntegrationRestException;
+
+import okhttp3.HttpUrl;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+public class ChannelRestFactory {
```

Comment:
How is this entire file new? I thought this already existed?

Reply:
The other class is rest connection, this new file is just rest functionality